build-multiarch/build-scripts/get-kustomize.sh

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
Why empty here? Forgot something?

Reply:
put all download operation into get-helm.sh by mistake, going to fix it